### ISSUE TITLE
IT Services - Status

### ISSUE THREAD

**Issue by DuboisJeremy** [alexanderzobnin/grafana-zabbix#199]
Hi everyone,

Here is my issue. I need to use Grafana-Zabbix to display IT Services. The goal is to get the "Status" (Ok, Warning, etc...) and display it in a Single Stat Panel.

But when I chose one IT Service and select status I get a number (0 for OK and 2 for Warning). First I don't know if the problem come from Zabbix or Grafana but it seems weird that I don't get the original text.

Even if Zabbix send number, I tried to treat those data with the "Value to text mapping" saying that if this is "2" then I got "Warning" and if I get "0" then I got "OK". The problem is that the status is not an integer but a string... So I can't use the "Value to text mapping" or even the coloring option...

I wanted to know if someone has the same issue, and a little request, is it possible to make a coloring for string/text (if text = "OK" then "Backgroung" is green.

Thanks for your help.

Regards

Jeremy

**Comment by alexanderzobnin** [comment]
Hi! Which plugin version you use?

**Comment by alexanderzobnin** [comment]
This bug was fixed in version 3.0 (now in beta, compatible with Grafana 3.0 only). So if you use old version, you need to switch to new.

**Comment by DuboisJeremy** [comment]
Indeed we are in 2.6. We migrated to 3.0 Beta but we rolled back due to Template issue (before your last update) and I didn't try SLA in this version.

I'll try again when we will migrate again in 3.0.

Thanks for your answer.

**Comment by DuboisJeremy** [comment]
Hi again,

We migrated to Grafana 3.0.2 and Zabbix plugin is 3.0 beta 2. But the problem is still here...

The first time I tried a quick workaround that consisted in setting Unit to "Short". It worked fine and the string "0" became a number and "Value to text mapping" was working. But know, when I do this, the status no longer become a number.

I don't know if the issue comes from the plugin or me but I will enjoy to find the reason !

Thanks for your help.

Regards

Jeremy
